.github/workflows/scripts/css_check.py

```
@@ -0,0 +1,350 @@
+    color_code_pattern = r"#(?:[0-9a-fA-F]{3}|[0-9a-fA-F]{6})\b"
+    violations = []
+
+    for line_number, line in enumerate(content.splitlines(), start=1):
+        matches = re.findall(color_code_pattern, line)
+        for match in matches:
+            violations.append((line_number, match))
+
+    return violations
+
+
+def process_typescript_file(
```

Ruff (0.12.2)

Missing return type annotation for public function `process_typescript_file`

Add return type annotation: `None`

(ANN201)